src/pages/add-lock-exception/AddLockExceptionForm/OrganisationUnitSelection/use-org-unit-cache.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+
+const useOrgUnitCache = () => {
+    const [cache, setCache] = useState(new Map())
+
+    return {
+        has: (currentRootId = null, itemId) =>
+            cache.has(currentRootId) && cache.get(currentRootId).has(itemId),
+        set: (currentRootId = null, itemId, orgUnits) => {
```

Comment:
`itemId` will either be the `groupId` or `levelId` used by the selection component.